bindings/node/lib/bindings/tokenizer.js

```diff
@@ -1,6 +1,11 @@
 const native = require("./native");
 
+class Tokenizer extends native.tokenizer_Tokenizer {
```

Comment:
@n1t0 I went for something very simple, I was not completely sure at first that it would work but it does 😁.
Let me know if it's good for you, I might add more JS tests early next week, but feel free to merge without them!

Reply:
That seems great! Thank you @Pierrci!